helpers/paragraph.js

```diff
@@ -142,16 +138,12 @@ export function calcSmartPrependPoint(note: TNote): number {
  * @param {string} paraText - the text to prepend
  * @param {ParagraphType} paragraphType - the usual paragraph type to prepend
  */
-export function smartPrependPara(
-  note: TNote,
-  paraText: string,
-  paragraphType: ParagraphType,
-): void {
+export function smartPrependPara(note: TNote, paraText: string, paragraphType: ParagraphType): void {
```

Comment:
When a func definition has more than a couple of parameters, following nmn's lead, I lay them out this way to make it easier to read.

Reply:
TBH, I didn't (intentionally) make these formatting changes, VSCode/Prettier/.eslint must have reformatted on save. I don't know how to tell it not to. Do you or @mikeerickson? Are you using the standard .eslintrc settings in the repo?

Reply:
Aha.
I appear to have prettier turned off. I guess I got fed up of it changing things like this where occasionally I find particular layouts helpful.
I think I'm using the standard .eslintrc file.